health/service.go

```diff
@@ -72,15 +72,10 @@ func hostName() string {
 func m3ctlHealthCheck(w http.ResponseWriter, r *http.Request) {
 	start := time.Now()
 	host := hostName()
-	h := healthCheckResult{Host: host, Timestamp: start}
 	status := healthCheck()
+	h := healthCheckResult{Host: host, Timestamp: start, Status: status}
 	h.ResponseTime = time.Since(start)
 
-	h.Status = status
```

Comment:
There is a slight change to the functionality here where if there is a !ok status we return a 200 with the status as part of the body. Personally, I think this is helpful because you can convey information like being in a bootstrapping state, where the server is up but not ready to receive requests. It is also the case that we hardcode the status to be `ok` which means that under no circumstances can we return a 500 here anyway.